### ISSUE TITLE
Add seeAlso link to IIIF Manifests

### ISSUE THREAD

**Issue by mikeapp** [yalelibrary/YUL-DC#1375]
**Story**

IIIF Manifests may contain a `seeAlso` property that links to a structured metadata file.   We'd like to link to the MODS XML records that are published from the OAI provider that was added to the site.

Example:

```
"seeAlso":
    {
      "@id": "https://collections.library.yale.edu/catalog/oai?verb=GetRecord&metadataPrefix=oai_mods&identifier=oai:collections.library.yale.edu:3",
      "format": "application/mods+xml",
      "profile": "http://www.loc.gov/mods/v3"
    }
```

**Acceptance**
- [ ] The `seeAlso` property is included in Manifests

IIIF Presentation spec:  https://iiif.io/api/presentation/2.1/#seealso

**Comment by FCRodriguez7** [comment]
PR: https://github.com/yalelibrary/yul-dc-management/pull/675

**Comment by FCRodriguez7** [comment]
Released management v2.45.3
